lib/styles/settings/_radiuses.scss

```diff
@@ -2,3 +2,5 @@ $radius-xs: 2px;
 $radius-s: 4px;
 $radius-m: 6px;
 $radius-l: 8px;
+//Special radius for components which have to be always rounded
+$radius-xl: 200px;
```

Comment:
should be as `xl`? maybe more specific? what if we want to add a border, e.g. 12 px as xl?

Reply:
https://bethinkteam.slack.com/archives/C012R8RM3A4/p1707753780620249?thread_ts=1707753172.775679&cid=C012R8RM3A4
It is the same as in figma